packages/ownable/src/lib.rs

```diff
@@ -202,89 +346,11 @@ impl<T: AddressLike> Ownership<T> {
     }
 }
 
-fn none_or<T: Display>(or: Option<&T>) -> String {
+// This is a nice helper, maybe move to dedicated utils package?
```

Comment:
Let's not expose this fn, there are other libraries out there that provide these kind of helpers.

Reply:
what libs provide this? I've been using this a lot.

Reply:
Simple enough to inline this logic where `cw-nfts` uses it at least: https://github.com/public-awesome/cw-nfts/pull/172/commits/6e3f86ff034ea22bc0c7b2ccff235e3d92760a69#diff-adb85ac50816403b5488531f2e35d16f88cad1b9a3f135e957866af708421d52L699